### DISCUSSION TITLE
Parse printing data option

### DISCUSSION THREAD

**Discussion by soaringdata** [sumatrapdfreader/sumatrapdf#3530] [Ideas]
It would be great if there was a setting to parse printing data before it goes to the driver
(i.e. 150 dpi)
This would allow printing to occur faster despite how a pdf file is authored/created and despite whatever printer is connected

**Comment by GitHubRulesOK** [comment]
the resolution is set by the print driver so MS controlled print to PDF is usually 600dpi but others you may set it to what you want
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/694b1d01-2592-4cc2-8b62-96a1178b9ebc)
